charts/rancher-webhook/templates/webhook.yaml

```diff
@@ -17,3 +17,23 @@ webhooks:
   name: rancher.cattle.io
   sideEffects: None
   timeoutSeconds: 10
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: rancher.cattle.io
+webhooks:
+- admissionReviewVersions:
+  - v1
+  - v1beta1
+  clientConfig:
+    service:
+      name: rancher-webhook
+      namespace: cattle-system
+      path: /v1/webhook/validation
+      port: 443
+  failurePolicy: Ignore
+  matchPolicy: Equivalent
+  name: rancher.cattle.io
+  sideEffects: None
+  timeoutSeconds: 10
```

Comment:
The webhooks list is optional.
``` suggestion
```

Reply:
I'm trying to match what the chart declares for the "ValidatingAdmissionWebhook", which this does.

So my options are:
- Both have this irrelevant value
- Only one has the irrelevant value (don't like this)
- Both don't have the irrelevant value

What is your preference?

Reply:
I'd prefer that the solutions match-up.  I wonder if there was a reason for including the irrelevant value in validating?

Reply:
We should remove the values from both if we want them to be the same. Especially since it can be misleading if someone doesn't know they get overwritten on startup.

Reply:
Yeah, that would probably be best (remove from both)

Reply:
I've removed the `webhooks:` portion from both.